docs/src/docs/core/FileButton.mdx

```diff
@@ -23,3 +23,9 @@ import { FileButtonDemos } from '@mantine/demos';
 Set `multiple` prop to enable picking multiple files:
 
 <Demo data={FileButtonDemos.multiple} />
+
+## Reset File
+
+You can use the `resetRef` to reset the currently selected file:
```

Comment:
Change to:

`resetRef` should be used to fix issue with stale value on hidden input element as input type file cannot be controlled. Call `resetRef` when user selection is cleared: